tracee-rules/output.go

```diff
@@ -12,11 +13,36 @@ import (
 	"github.com/aquasecurity/tracee/tracee-rules/types"
 )
 
-func setupOutput(webhook string) (chan types.Finding, error) {
+const DetectionOutput string = `
+*** Detection ***
+Time: %d
```

Comment:
can you please format as time?

Reply:
I'm printing Unix time which is an int64, do you want a different format instead?

Reply:
Anything human readable, don't you think?

Reply:
OK I can change it to a more conventional format.

Reply:
I changed to RFC3339 https://github.com/aquasecurity/tracee/pull/573/commits/29873e935ae6b87a46f2281863d0546e51699737